assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -3,83 +3,29 @@
  */
 import { useDispatch } from '@wordpress/data';
 import { Modal } from '@wordpress/components';
-import { useEffect, useLayoutEffect, useState } from '@wordpress/element';
-import { store as blockEditorStore } from '@wordpress/block-editor';
+import { useState } from '@wordpress/element';
 import { store as editorStore } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import Wizard from './wizard';
 import useEditorWizardSteps from './use-editor-wizard-steps';
+import { useWizardOpenState, useDefaultTemplate } from './helpers';
 import '../../shared/data/api-fetch-preloaded-once';
 
-/**
- * A React Hook to observe if a modal is open based on the body class.
- *
- * @param {boolean} shouldObserve If it should observe the changes.
- *
- * @return {boolean|undefined} Whether a modal is open, or `undefined` if it's not initialized yet.
- */
-const useObserveOpenModal = ( shouldObserve ) => {
```

Comment:
To simplify a little this component file, I moved these hooks to a `helpers.js` file.